app/controllers/concerns/verify_sp_attributes_concern.rb

```diff
@@ -62,6 +62,16 @@ def verified_after_consent?(last_estimated_consent)
     verification_timestamp.present? && last_estimated_consent < verification_timestamp
   end
 
+  def linked_identity_ial
+    if resolved_authn_context_result.ialmax?
+      0
+    elsif resolved_authn_context_result.identity_proofing?
+      2
+    else
+      1
+    end
```

Comment:
We have another story for sending the VoT back to the partner. @theabrad is working on it. For that we'll likely need to add the `acr_values` and `vtr` to the identity column and start reading from those instead of the `ial` and `aal` columns. Once we do that we can remove this portion entirely.